feature/deps.go

```diff
@@ -58,6 +58,7 @@ var deps = depDesc{
 	lnwire.AnchorsOptional: {
 		lnwire.StaticRemoteKeyOptional: {},
 	},
+	lnwire.AnchorsZeroFeeHtlcTxOptional: {},
```

Comment:
Remove optional signal as well? Can defer perhaps until the rc phase as well

Reply:
This is just the dependency check, I don't think it affects the signalling, @cfromknecht ?

Reply:
yeah this can be removed, this is equivalent to not having an entry in the dep map

Reply:
Ok, removed! Should we remove the `lnwire.AnchorsOptional` entry also?